examples/er-rest-example/project-conf.h

```diff
@@ -93,6 +96,13 @@
 #undef COAP_PROXY_OPTION_PROCESSING
 #define COAP_PROXY_OPTION_PROCESSING   0
 
+/* Turn of DAO ACK to make code smaller */
+#undef RPL_CONF_WITH_DAO_ACK
+#define RPL_CONF_WITH_DAO_ACK          0
```

Comment:
All examples are still sky based (e.g. simulations) - but I guess I can do a quick test and see if I can fix that also.